WalletWasabi/WabiSabi/Backend/Rounds/CoinJoinStorage/CoinJoinTransactionArchiver.cs

```diff
@@ -50,6 +53,50 @@ public async Task<string> StoreJsonAsync(Transaction transaction, DateTimeOffset
 		return filePath;
 	}
 
+	public IEnumerable<TransactionInfo> ReadJson(DateTimeOffset from, DateTimeOffset to)
+	{
+		if (to < from)
+		{
+			throw new ArgumentException("Invalid time period", nameof(from));
+		}
+
+		List<string> dirsToRead = new();
+		int monthOffset = 0;
+
+		do
+		{
+			DateTimeOffset date = new(from.Year, from.Month + monthOffset, from.Day, 0, 0, 0, from.Offset);
+			if (date > to)
+			{
+				break;
+			}
+			dirsToRead.Add(Path.Combine(BaseStoragePath, $"{date:yyyy-MM}"));
```

Comment:
`yyyy-MM` should be a constant. Then the two places where it is used would be easily spotted.